src/EhrgoHealth.WebService/Models/AllergyIntolerance.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Rest;
+
+namespace EhrgoHealth.WebService.Models
+{
+    public class AllergyIntolerance
+    {
+        //Dictonary looks up possible allergies for a given medication. Example allergyLookup<Medication, List<Codes>>
+        private Dictionary<string, List<string>> allergyLookup = new Dictionary<string, List<string>>();
+        private FhirClient fhirClient = null;
+
+
+        public AllergyIntolerance(string fhirServer)
+        {
+            if (fhirServer == null || fhirServer == String.Empty)
+                throw new Exception("Invalid URL passed to AllergyIntolerance Constructor");
+
+            fhirClient = new FhirClient(fhirServer);
+
+            //We do not use a database for the webservice. We will hardcode a handful of allergies and their intolerances

```

Comment:
this is perfectly valid at this stage of a prototype...mind moving the data to a constants file/class